### PR TITLE
Allow multiple root elements in vue-component slot

### PR DESCRIPTION
**Pull request by larenelg**
Fixes https://github.com/ngVue/ngVue/issues/113

### CODE REVIEW

File: example/vue-component-slots/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      <div class="input-field col s12">
+        <textarea class="materialize-textarea"
+                  arial-label="person description"
+                  ng-model="ctrl.person.description"></textarea>
+      </div>
+    </div>
+  </div>
+  <script src="/assets/vue-component-slots.build.js"></script>
+</body>
+</html>
```
Comment (KodrAus) [inline example/vue-component-slots/index.html:70]: Looks like we've got a missing trailing newline here

**Comment by jaredmcateer** [comment]
@larenelg Sorry for letting this sit for so long I hadn't set up this repo for watching I'll try to take a look at this PR this today or tomorrow. Thank you for your contribution.

**Comment by larenelg** [comment]
Thanks @jaredmcateer! I just got back from leave so will respond to feedback promptly :)

**Comment by jaredmcateer** [comment]
RE: status of slots in your issue associated with this PR, nothing hasn't changed. It's really risky placing angular code inside of them and is basically do at your own risk. There isn't much active development on this package as the original creators have seemed to have moved on, I obtained member status so I could publish bug fixes. I'm happy to incorporate your changes though.

**Comment by larenelg** [comment]
Hey @jaredmcateer. RE: slots, yep, totally makes sense. After a few weeks of dev I think we're pretty convinced that only using self-contained Vue views/components with no slots is the way to go :) So, no real need for this change to be merged. Thanks for your review!

**Pull request closed without merging**